### PR TITLE
Add privacy policy

### PR DESCRIPTION
**Pull request by Oaphi**
This PR adds a page for the privacy policy as well as the policy itself. The update includes:

- new /privacy route, layout, and view;
- split of application and dark mode client-side JS (as the privacy page only needs the switch);

### CODE REVIEW

File: views/layouts/application.ejs [hunk 1]
```diff
@@ -22,6 +22,7 @@
         integrity="sha384-JjSmVgyd0p3pXB1rRibZUAYoIIy6OrQ6VrjIEaFf/nJGzIxFDsf4x0xIM+B07jRM"
         crossorigin="anonymous"></script>
     <script type="application/javascript" src="/javascripts/application.js?1"></script>
```
Comment (mousetail) [inline views/layouts/application.ejs:24]: You probably want to bump the version for application.js here too

File: views/layouts/application.ejs [hunk 2]
```diff
@@ -43,6 +44,9 @@
                     <li>
                         <a href="/#faq">FAQ</a>
                     </li>
+                    <li>
+                        <a href="/privacy">Privacy</a>
```
Comment (mousetail) [inline views/layouts/application.ejs:48]: I wouldn't put this so prominently in the title, can you find a place in the footer maybe?

File: views/layouts/application.ejs (hunk 1 above)
Comment (Oaphi) [reply views/layouts/application.ejs]: Yup, done, spaced out on that. I'll try to come up with something that automates cache breaking, it's easy to forget to do.

File: views/layouts/application.ejs (hunk 2 above)
Comment (Oaphi) [reply views/layouts/application.ejs:48]: No problem, will do - I need to address a styling issue with the footer either way.

**Pull request merged**